### DISCUSSION TITLE
How to navigate through files like in the homepage video?

### DISCUSSION THREAD

**Discussion by RichardBray** [kovidgoyal/kitty#5019] [Q&A] [answered]
Hello there,

From the video, on the kitty homepage, https://sw.kovidgoyal.net/kitty/ around the 3:25 mark Kovid is able to click and open files with ctrl+shift+click.

I have kitty installed on my mac and I am unable to do that. Is there something I need to install?

**Comment by kovidgoyal** [comment, marked as answer]
https://sw.kovidgoyal.net/kitty/open_actions/

**Reply by RichardBray** [reply]
Oh sweet, thanks 👍

**Reply by RichardBray** [reply]
Hmm, maybe hyperlink support isn't there yet for macs 🤷 
```
➜ ls --hyperlink=auto
ls: unrecognized option `--hyperlink=auto'
usage: ls [-@ABCFGHILOPRSTUWabcdefghiklmnopqrstuvwxy1%,] [--color=when] [-D format] [file ...]
```

**Reply by RichardBray** [reply]
actually, I think hyperlinks should be on by default on Macs 🤔

**Reply by page-down** [reply]
You need to install GNU Coreutils and use `gls` (or make an alias).

https://www.gnu.org/software/coreutils/

https://formulae.brew.sh/formula/coreutils

Or use any other program that supports output hyperlink.

**Comment by kovidgoyal** [comment]
If hyperlinks are output then hovering your mouse over the filenames
will cause them to be underlined indicating they are clickable.
